lib/applications.js

```diff
@@ -624,9 +624,11 @@ exports.postApplication = async (req, res) => {
 
         // We don't need to recalculate the votes amount, as the pax type is not set here.
 
+        const notificationEmail = await core.getMember(req, newApplication.user_id).notification_email;
```

Comment:
if it's the same user as the one who posts the application, you can just as well use `req.user.notification_email`, no need for extra request here

Reply:
True, will fix this